### DISCUSSION TITLE
Config help

### DISCUSSION THREAD

**Discussion by diniamo** [marlonrichert/zsh-autocomplete#607] [Q&A]
I want to set 2 things:
1. I want history and Ctrl + R to match exact strings, so `rg` should not be completable to `xrandr --output HDMI-1 --brightness 1` (example from my history)
2. When using history or Ctrl + R, I want my left and right arrows keys to move in the prompt and not the completion.

Additionally, how can I go further than the set limit in Ctrl + R search?

These have probably been asked before, I have no clue what to search for though.

**Comment by marlonrichert** [comment]
1. That's not yet configurable. Feel free to open a feature request for that.
2. That's unfortunately not possible without also changing this in the completion menu. However, if that doesn't bother you, you can do the following:
   1. Run `cat -v` and then press the arrow keys to see which escape sequences your terminal outputs for them.
   2. Add to your `.zshrc` file _after_ sourcing Autocomplete:
   ```sh
   bindkey -M menuselect '<left arrow escape>' .backward-char
   bindkey -M menuselect '<right arrow escape>' .forward-char
   ```

**Reply by JerryZhongJ** [reply]
This is what I add to the end of `.zshrc`:
```
bindkey -M menuselect '^[[D' .backward-char
bindkey -M menuselect '^[[C' .forward-char
```
It doesn't work.
